iocore/net/SSLStats.cc

```diff
@@ -202,6 +202,10 @@ SSLInitializeStatistics()
   RecRegisterRawStat(ssl_rsb, RECT_PROCESS, "proxy.process.ssl.ssl_ocsp_refresh_cert_failure", RECD_INT, RECP_PERSISTENT,
                      (int)ssl_ocsp_refresh_cert_failure_stat, RecRawStatSyncCount);
 
+  /* TLSv1.3 0-RTT stats */
+  RecRegisterRawStat(ssl_rsb, RECT_PROCESS, "proxy.process.ssl.ssl_early_data_received", RECD_INT, RECP_PERSISTENT,
```

Comment:
Don't dupe "ssl" here in the name space. I'd prefer e.g. proxy.process.ssl.early_data_received

Reply:
Done.